### DISCUSSION TITLE
Temperature coupling in phase field simulation of dendrite growth

### DISCUSSION THREAD

**Discussion by biaogxb** [idaholab/moose#21805] [Q&A Modules: Phase field] [answered]
I want to simulate the growth of lithium dendrite with phase field and study the influence of temperature on the growth morphology of dendrite. Through the literature, I know that I can couple the temperature with the diffusion rate of lithium ions. Is there any example about the influence of temperature on dendrite growth?
![image](https://user-images.githubusercontent.com/95063954/183611878-dc2dadab-dbc6-4152-95b9-ad0a8a7b183c.png)
![image](https://user-images.githubusercontent.com/95063954/183611943-759ea2ff-a24d-45f5-8e09-9d07c96f211c.png)

Looking forward to your reply, thank you!

**Comment by lindsayad** [comment, marked as answer]
@dschwen could you possibly help with this question?

**Reply by SudiptaBiswas** [reply]
We recently published an article that systematically analyzes the effect of different parameters during alloy solidification (it includes the temperature effect as well). You might find it useful. 
https://iopscience.iop.org/article/10.1088/1361-651X/ac46dc/meta

**Reply by biaogxb** [reply]
Thank you for your reply. I will study this article carefully!

**Reply by lindsayad** [reply]
thank you @SudiptaBiswas !
